src/components/forms/Forms.tsx

```diff
@@ -0,0 +1,13 @@
+import { Stack } from '@chakra-ui/react';
+import { FC } from 'react';
+import { useRouter } from 'next/router';
+
+import { ProjectGrantsForm } from './ProjectGrantsForm';
+
+import { PROJECT_GRANTS_APPLY_URL } from '../../constants';
+
+export const Forms: FC = () => {
+  const router = useRouter();
+
+  return <Stack>{router.pathname === PROJECT_GRANTS_APPLY_URL && <ProjectGrantsForm />}</Stack>;
```

Comment:
Wondering why you choose to abstract the forms selection here instead of just rendering them inside each `apply.ts` page which makes it simpler.

```ts
// inside the apply.ts you could print the corresponding form

const ProjectGrantsApply: NextPage = () => {
  return (
    <>
      ...

      <ProjectGrantsForm />
    </>
  );
};
```

Reply:
This is related to the [mobile layout](https://www.figma.com/file/s5JQ7HOyKOWVJCf5mkx3xO/ESP-Website-2021?node-id=1253%3A1488). The pages use the layout which applies padding and other styles. In mobile version, forms are out of that layout, if I include them in the apply pages I'd need to override the layout some way, making the css more complex, I think.

Reply:
Got it, make sense 👍🏼